project.go

```diff
@@ -0,0 +1,392 @@
+// Copyright 2016 EF CTX. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/tsuru/gnuflag"
+	"github.com/tsuru/tsuru/cmd"
+	tsuruerrors "github.com/tsuru/tsuru/errors"
+)
+
+type projectCreate struct {
+	fs       *gnuflag.FlagSet
+	name     string
+	platform string
+	team     string
+	plan     string
+	envs     commaSeparatedFlag
+}
+
+func (*projectCreate) Info() *cmd.Info {
+	return &cmd.Info{
+		Name: "project-create",
+		Desc: "creates a remote project in the tranor server",
+	}
+}
+
+func (c *projectCreate) Run(ctx *cmd.Context, client *cmd.Client) error {
+	if c.name == "" || c.platform == "" {
+		return errors.New("please provide the name and the platform")
+	}
+	config, err := loadConfigFile()
+	if err != nil {
+		return errors.New("unable to load environments file, please make sure that tranor is properly configured")
+	}
+	err = c.envs.validate(config.envNames())
+	if err != nil {
+		return fmt.Errorf("failed to load environments: %s", err)
+	}
+	envs := c.filterEnvironments(config.Environments, c.envs.Values())
+	apps, err := c.createApps(envs, client)
```

Comment:
If someone tried to create a project with the same name but in different environments, it seems tranor would allow it, wouldn't it? Eg:
```
tranor project-create --name xubiru --envs dev
tranor project-create --name xubiru --envs qa
```
Perhaps we want to deal with it somehow.